cmd/uplink/cmd/cat.go

```diff
@@ -28,7 +28,7 @@ func catMain(cmd *cobra.Command, args []string) (err error) {
 
 	ctx := process.Ctx(cmd)
 
-	u0, err := utils.ParseURL(args[0])
+	u0, err := fpath.New(args[0])
```

Comment:
Some better name like: `infile`, `input`, `src`

Reply:
Changed it everywhere in all commands.